ultimatejobweb/templates/sign_up.html

```diff
@@ -83,6 +88,7 @@ <h1 class="display-3">UltimateJob</h1>
     </div>
 </div>
 
+<footer>
```

Comment:
should that be /footer?

Reply:
Yes, i used the footer tag and added a css file to make the footer stay at the bottom of the page, at the same spot in all pages

Reply:
I see what you mean now, yes i didn't add the / to close the footer tag, silly mistake..